config/nim.cfg

```diff
@@ -182,7 +182,6 @@ gcc.maxerrorsimpl = "-fmax-errors=3"
 @end
 
 @if macosx or freebsd or openbsd:
-  cc = clang
```

Comment:
But this affects the existing OSX and BSD targets!

Reply:
@Araq Thank you for pointing this out! I will fix that. Of course, BSD should not be affected in any way.

Give me few days please, I am just back to Taiwan and building gcc12 across several machines. I will deal with this and csources PR later this week, I hope.